src/components/ModalAlertNotSupported.js

```diff
@@ -19,6 +19,7 @@ import ModalAlert from './ModalAlert.js';
 class ModalAlertNotSupported extends React.Component {
   render() {
     const renderBody = () => {
+      if (this.props.body) return this.props.body;
```

Comment:
We already had this in `this.props.children`. Was it not enough?

```
return (
      <ModalAlert
        id="notSupported"
        title={this.props.title || t`Action not supported`}
        buttonName={t`Close`}
        body={this.props.children || renderBody()}
      />
    );
```

Reply:
Did not see the children, changed to use it and removed the body prop